tools/dev.py

```diff
@@ -0,0 +1,180 @@
+#! /usr/bin/python3
+# -*- coding: utf-8 -*-
+
+import argparse as argp
+from os.path import dirname, normpath
+from shlex import quote
+import subprocess as subp
+
+C_blue = "\033[1;34m"
+C_purple = "\033[1;35m"
+C_yellow = "\033[1;33m"
+C_white = "\033[1;37m"
+C_normal = "\033[0m"
+
+PROJECT_NAME = "zosia"
+ROOT_DIR = normpath(dirname(__file__) + "/..")
+DOCKER_COMPOSE = f"{ROOT_DIR}/docker-compose.dev.yml"
+WEB_CONTAINER_NAME = f"{PROJECT_NAME}_web_1"
+DB_CONTAINER_NAME = f"{PROJECT_NAME}_db_1"
+FILE_SYSTEM_NOTE = f"({C_yellow}note:{C_normal} this may create files on host fs with root permissions)"
+
+
+def shell_run(command):
+    if args.debug:
+        print(f"{C_white}** {C_yellow}{command}{C_white} **{C_normal}")
+
+    subp.run(command, shell=True)
```

Comment:
`shell=True` should not be used. Please consider using other python functions to get same results, like
`glob`, `fnmatch`, `os.walk()`, `os.path.expandvars()`, `os.path.expanduser()`, and `shutil`

Reply:
I suppose we can get rid of `shell=True`, because `docker` and `docker-compose` are recognized as an executable.